src/msgpack/Convertes/ArrayConverter.cs

```diff
@@ -1,7 +1,7 @@
 using System;
 using System.Collections.Generic;
 
-namespace TarantoolDnx.MsgPack
+namespace TarantoolDnx.MsgPack.Convertes
```

Comment:
Current files moving decided to leave as is.
Future project re-organizations will be approved.